pastastore/plotting.py

```diff
@@ -949,21 +963,33 @@ def add_labels(df, ax, **kwargs):
             DataFrame containing x, y - data. Index is used as label
         ax: matplotlib.Axes
             axes object to label points on
+        adjust: bool
+            automated smart label placement using adjustText
         **kwargs:
             keyword arguments to ax.annotate
         """
-
         stroke = [patheffects.withStroke(linewidth=3, foreground="w")]
 
-        fontsize = kwargs.pop("fontsize", 10)
-        textcoords = kwargs.pop("textcoords", "offset points")
-        xytext = kwargs.pop("xytext", (10, 10))
-
-        for name, row in df.iterrows():
-            namestr = str(name)
-            txt = ax.annotate(text=namestr,
-                              xy=(row["x"], row["y"]),
-                              fontsize=fontsize,
-                              textcoords=textcoords,
-                              xytext=xytext)
-            txt.set_path_effects(stroke)
+        if adjust:
+            texts = []
+            for name, row in df.iterrows():
+                texts.append(ax.text(row['x'], row['y'], str(name),
+                                     **{'path_effects': stroke}))
+            adjust_text(texts, force_text=0.05, arrowprops=dict(
```

Comment:
does adjust_text also take patheffects as a keyword argument? If so would be nice to put it in here, or maybe set them on the resulting annotation labels.

Reply:
I tried passing the path_effects as both *args and **kwargs to the `adjust_text `function but it does not seem to actually plot it. Passing it to the `ax.text()` currently works so maybe just stick to that?

Reply:
As in, passing it to `ax.text` also results in the newly adjusted annotates to have the patheffects as well? In that case, that sounds like a good solution.

Reply:
Yes passing with passing it to `ax.text()` you get the result as shown in the second figure of this PR.